Examples/Algorithms/Propagation/include/ACTFW/Propagation/PropagationAlgorithm.ipp

```diff
@@ -169,16 +169,16 @@ ProcessCode PropagationAlgorithm<propagator_t>::execute(
     PropagationOutput pOutput;
     if (charge) {
       // charged extrapolation - with hit recording
-      Acts::BoundParameters startParameters(context.geoContext, std::move(cov),
-                                            std::move(pars), surface);
-      sPosition = startParameters.position();
+      Acts::BoundParameters startParameters(surface, std::move(pars),
+                                            std::move(cov));
+      sPosition = startParameters.position(context.geoContext);
       sMomentum = startParameters.momentum();
```

Comment:
I agree, this makes it really clear: the `Context` object is used when it is needed and not at construction of the `BoundParameters`.